src/provider/camunda-platform/properties/ImplementationTypeProps.js

```diff
@@ -238,6 +246,16 @@ function UpdateModdlePropertiesCommand(element, businessObject, newProperties) {
   };
 }
 
-function sortByName(options) {
-  return sortBy(options, o => o.label.toLowerCase());
+function sortByPriority(options) {
```

Comment:
Can you maybe explain why do we need this, instead of simply ordering them above? Probably because the options can be added dynamically?